### PR TITLE
Fix release schedule link in release and patch initial checklist

### PR DESCRIPTION
**Pull request by tarunvijwani**
We moved the [release schedule page](PdToLP-K-p2) to [WC Blocks repo](dToLP-p2). This PR updates the release schedule link present in the release and patch initial checklist.

### CODE REVIEW

File: .github/patch-initial-checklist.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Additionally, make sure to differentiate between things in the testing notes tha
   * If the base branch was `trunk`, and this release was deployed to WordPress.org, then merge the branch into `trunk`.
   * If the base branch is the release branch this patch release is for, then merge branch into release branch and then merge the release branch back to `trunk` if the patch release is the latest released version. Otherwise just merge back into the release branch.
 * [ ] If you merged the branch to `trunk`, then update version on the `trunk` branch to be for the next version of the plugin and include the `dev` suffix (e.g. something like [`2.6-dev`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/commit/e27f053e7be0bf7c1d376f5bdb9d9999190ce158)) for the next version.
-* [ ] Update the schedules p2 with the shipped date for the release (Pca54o-1N-p2).
```
Comment (alexflorisca) [inline .github/patch-initial-checklist.md:66]: It's good to use a tampermonkey script here as we ideally don't want internal urls to be public

File: .github/release-initial-checklist.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ Additionally, make sure to differentiate between things in the testing notes tha
     * [ ] `readme.txt`
     * [ ] `src/Package.php`
     * [ ] `woocommerce-gutenberg-products-block.php`.
-* [ ] Update the schedules p2 with the shipped date for the release (Pca54o-1N-p2).
+* [ ] Update the schedules p2 with the shipped date for the release ([`Release Schedules`](https://wooblocksrepo.wordpress.com/release-schedules/)).
```
Comment (alexflorisca) [inline .github/release-initial-checklist.md]: Same as above ^^

**Review by alexflorisca — CHANGES_REQUESTED** [review]
Hey @tarunvijwani, ideally we would want to use a TamperMonkey script to hide any internal links. See this FG page (PCYsg-5Xx-p2) and the [TM script here](https://github.com/Automattic/tampermonkey-highlighter)

**Comment by tarunvijwani** [comment]
> Hey @tarunvijwani, ideally we would want to use a TamperMonkey script to hide any internal links. See this FG page (PCYsg-5Xx-p2) and the [TM script here](https://github.com/Automattic/tampermonkey-highlighter)

Thank you, @alexflorisca! It was a TIL for me. I have added the TamperMonkey script generated link now. 🙂

Link added: PdToLP-K-p2

**Review by alexflorisca — APPROVED** [review]
Looks good 👍

**Pull request merged**